examples/src/test/resources/examples/datasource_many_dynamictables_legend/legend.jrxml

```diff
@@ -4,7 +4,7 @@
 	<property name="ireport.zoom" value="2.0"/>
 	<property name="ireport.x" value="0"/>
 	<property name="ireport.y" value="0"/>
-	<style name="Default" isDefault="true" fontName="Arial"/>
+	<style name="Default" isDefault="true" fontName="DejaVu"/>
```

Comment:
Why not "Liberation Sans"? It would be good if we could use a single font for all tests.